01_AppCore/Business/Models/Results/ExceptionResult.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+namespace _01_AppCore.Business.Models.Results
+{
+    public class ExceptionResult : Result
+    {
+        public ExceptionResult(Exception exception, bool showException = true)
+            : base(ResultStatus.Exception,
+                  showException ?
+                    (exception != null ?
+                        "Exception: " + exception.Message + (exception.InnerException != null ?
+                            " | Inner Exception: " + exception.InnerException.Message + (exception.InnerException.InnerException != null ?
+                                " | " + exception.InnerException.InnerException.Message
+                            : "")
+                        : "")
+                    : "")
+                  : "")
+        {
+
+        }
+
+        public ExceptionResult() : base(ResultStatus.Exception, "")
+        {
+
+        }
+    }
+
+    public class ExceptionResult<TResultType> : Result<TResultType>
+    {
+        public ExceptionResult(Exception exception, bool showException = true)
+            : base(ResultStatus.Exception,
+                  showException ?
```

Comment:
There can be even more exceptions here. I'd go over them in a loop add their messages together.